infra/libfuzzer-pipeline.groovy

```diff
@@ -90,11 +90,10 @@ def call(body) {
                         sh "mkdir -p $junit_reports"
                         stage("$sanitizer sanitizer ($engine)") {
                             // Run image to produce fuzzers
-                            def env = "-e SANITIZER=\"${sanitizer}\" "
+                            def env = "-e SANITIZER=\"${sanitizer}\" -e FUZZING_ENGINE=\"${engine}\" "
```

Comment:
Can you create a doc bug, we would need to update documentation with this flag, also need updating in helper.py.